backend/curriculum_tracking/tests/test_api_views.py

```diff
@@ -116,8 +116,16 @@ def setUp(self):
         self.assertEqual(self.card_1.status, AgileCard.IN_PROGRESS)
         self.assertEqual(self.card_2.status, AgileCard.IN_PROGRESS)
 
-    def test_request_review_permissions_non_superuser_staff(self):
+        self.login_as_superuser()
+        request_review_url = f"{self.get_instance_url(self.card_2.id)}request_review/"
+        response = self.client.post(request_review_url)
+
+        self.assertEqual(response.status_code, 200)
 
+        self.card_2.refresh_from_db()
+        self.assertEqual(self.card_2.status, AgileCard.IN_REVIEW)
```

Comment:
It looks like the setup for this test is taking on too much responsibility.  
Ideally a setup should only have sanity checking assersions (if any assersians at all)

A setup is not a test. Test functions start with the word `test_`

Can this be refactored so that the setup has less responsibility? 

The code works, but this is a bad practice and it shouldn't turn into a bad habit.

Reply:
Sanity checking asserts are almost like documentation. They should pass unless the world has gone totally mad.